src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs

```diff
@@ -730,11 +630,9 @@ private IList<ApprenticeshipListItemViewModel> MapFrom(IEnumerable<Apprenticeshi
         }
 
         // TODO: Move mappers into own class
-        private IEnumerable<CommitmentListItemViewModel> MapFrom(List<CommitmentListItem> commitments, bool showProviderMessage)
+        private IEnumerable<CommitmentListItemViewModel> MapFrom(IEnumerable<CommitmentListItem> commitments, bool showProviderMessage)
```

Comment:
There are mappers for other things (e.g. ApprenticeshipMapper). could the same thing apply for Commitments?

Reply:
I've switched the List-accepting params in this file to accept IEnumerable (where appropriate). We can handle other files in later refactors.